project/Build.scala

```diff
@@ -465,7 +463,7 @@ object Build extends sbt.Build {
             "-P:scalajs:fixClassOf",
             // Link source maps to github sources of original Scalalib
             "-P:scalajs:mapSourceURI:" +
-            fetchedScalaSourceDir.value.toURI +
+            (artifactPath in fetchScalaSource).value.toURI +
```

Comment:
Yes it could,  but that would fetch the sources if you say: `show scalacOptions`, which is conceptually wrong.